src/server/controlchan/control_loop.rs

```diff
@@ -198,7 +198,7 @@ where
                         };
                     },
                     _ = shutdown.listen() => {
-                        slog::info!(logger, "Shutting down control loop");
+                        slog::warn!(logger, "Closing open control connection because of shutdown signal");
```

Comment:
I'm not sure this should be a warning. This is the normal way the app ends.

Reply:
That's true... thought to do so since it's interrupting an open session. But it's debatable